content/en/developers/marketplace.md

```diff
@@ -108,36 +108,43 @@ Technology Partners can request access to the private Marketplace repository by
 
 The Datadog Development Toolkit command (`ddev`) allows you to create scaffolding when you are first developing your integration, by spinning up a skeleton of all the assets and metadata for your tile.
 
-Ensure that Python 3.8 or higher is installed.	
+Ensure that Python 3.8 or higher is installed (https://www.python.org/downloads/).	
 
-The latest released version may be installed from [PyPI][23]:
+To avoid a range of potential environment conflicts, we recommend going into the directory where you cloned the repository and creating a simple virtual environment with the following command: `python3 -m pip install virtualenv --user`
 
-`python -m pip install --upgrade "datadog-checks-dev[cli]`
+The latest released version of the Datadog Development Toolkit may then be installed from [PyPI][23]:
+
+`python -m pip install --upgrade "datadog-checks-dev[cli]"`
+
+**Note:** if you are using Z shell, you may need to use escape characters: `python -m pip install --upgrade datadog-checks-dev\[cli\]` 
 
 Set the location of to the cloned repository:
 
 #### Marketplace:
```

Comment:
I'd like to keep the headings for Marketplace and Integrations-Extras, to be very clear on the difference between the two. We've seen developers get confused on this point, since we are asking them to submit their integration to one or the other.